pkg/controller/combo_controller.go

```diff
@@ -153,6 +157,53 @@ func (c *combinationController) Reconcile(ctx context.Context, req ctrl.Request)
 		Message:            "evaluations successfully processed",
 	})
 
+	// If manifest application is indicated, create generated manifests on-cluster.
+	if combination.Spec.Apply {
+		// update status to indicate processing
+		combination.SetStatusCondition(metav1.Condition{
+			Type:               v1alpha1.TypeInProgress,
+			Status:             metav1.ConditionTrue,
+			Reason:             v1alpha1.ReasonProcessing,
+			LastTransitionTime: metav1.NewTime(time.Now()),
```

Comment:
We shouldn't be updating the `LastTransitionTime` unless the `Status` has changed since the last time we updated this condition. This kind of thing can result in a hotloop where you update the condition with this timestamp, and then reconcile because the status was updated and then update the transition time stamp again, and then reconcile again, etc.

Reply:
There are helpers for this in the apimachinery repo: https://github.com/kubernetes/apimachinery/blob/ec22c436956176895311ebeee513ca9dee0b3aa5/pkg/api/meta/conditions.go#L30

Reply:
Upon further inspection, it looks like `combination.SetStatusCondition` just calls that function under the hood. In that case, I think you can just drop all explicit setting of `LastTransitionTime`